napari/layers/image/image.py

```diff
@@ -218,6 +218,10 @@ def __init__(
         else:
             ndim = len(init_shape)
 
+        # Cast 3D images in float64 to float32
+        if ndim == 3 and data.dtype == np.float64:
```

Comment:
> Oh no, looks like I broke some tests... can't fix right now but will get to it ASAP

Ah i see why - let's do the down cast on the vispy side we actually have this line https://github.com/napari/napari/blob/d6bc683b019c4a3a3c6e936526e29bbd59cca2f4/napari/_vispy/vispy_image_layer.py#L89 and the fix data and dtype function here https://github.com/napari/napari/blob/d6bc683b019c4a3a3c6e936526e29bbd59cca2f4/napari/_vispy/utils_gl.py#L73 that you should be able to use. 

> uh oh, can't fix right now but will get to it ASAP!

No worries, take your time!